packages/remix-server-runtime/markup.ts

```diff
@@ -0,0 +1,16 @@
+// This escapeHtml utility is based on https://github.com/zertosh/htmlescape
```

Comment:
Could we get a comment here explaining why we're not just using that module?

Reply:
Also, I'm curious why that module hasn't simply been updated to improve its own performance 🤔

Reply:
I'm down to inline it personally. It's small and simple enough, and reducing deps in general is good IMO.

Reply:
Oh, I'm fine with that too. I just know it's a question I'd have if I was working in this code and I think a comment would help.

Reply:
i've added a comment [here](https://github.com/remix-run/remix/pull/3889/commits/3cc17531ecef87333e3b0611951796b28df538b3) :sweat_smile: